fable-test-app/src/pages/ViewHolidays.tsx

```diff
@@ -120,11 +120,11 @@ const ViewHolidays = () => {
 
       {provinceId === 'federal' ? (
         <Details detailsTitle="What are federal holidays?" open>
-          <Text marginBottom="0">If your job is regulated by the federal government, you get federal holidays instead of the provincial holidays. Find out more about <Link className="link-default" to="/federal-and-provincial-holidays">who gets federal holidays</Link>.</Text>
+          <Text marginBottom="0">If your job is regulated by the federal government, you get federal holidays instead of the provincial holidays. Find out more about <a className="link-default" href="/federal-and-provincial-holidays">who gets federal holidays</a>.</Text>
         </Details>
       ) : (
         <Details detailsTitle="What are optional holidays?" open>
-          <Text marginBottom="0">Optional holidays are commonly observed but not legally mandated. Businesses may choose to opt-in to optional holidays but they don't have to. Find out more about <Link className="link-default" to="/optional-holidays">optional holidays</Link>.</Text>
+          <Text marginBottom="0">Optional holidays are commonly observed but not legally mandated. Businesses may choose to opt-in to optional holidays but they don't have to. Find out more about <a className="link-default" href="/optional-holidays">optional holidays</a>.</Text>
```

Comment:
Removing the `Link` makes the app use the regular link, which forces a page refresh. It's fine in this case because everything other GCDS works that way in this app.

But, it was nice having this because it did function like a single page app. Another thing we could look at as well, is to `[<ScrollRestoration />](https://reactrouter.com/en/main/components/scroll-restoration)` but it does require you to use a data router and I don't think we currently are doing that in this app.